src/hooks/useENSRegistrationActionHandler.ts

```diff
@@ -164,6 +158,13 @@ export default function useENSRegistrationActionHandler(
         name.replace(ENS_DOMAIN, ''),
         duration
       );
+      const changedRecords = await uploadRecordImages(
```

Comment:
Wonder if we can call this in parallel with the previous `awaits`? Seems like it would cut a lot of the loading time off if we did.

Reply:
hm just moved it down from where it was in case the `loadwallet` or rent price failed

Reply:
i was also thinking there's this same issue when calculating gas limits, we use promise.all which could produce that if one of the promises fail everything fail and we default to the static default gas